config/locales/en.yml

```diff
@@ -1308,6 +1323,17 @@ en:
         You can only add one route from each question.
       </p>
   save_and_continue: Save and continue
+  secondary_skip:
+    new:
+      back: Back to question %{page_position}'s routes
+      caption: Question %{page_position}'s routes
```

Comment:
```suggestion
      back: Back to question %{page_position}’s routes
      caption: Question %{page_position}’s routes
```

Reply:
Thanks, I should really be better at checking these before asking for a review